packages/react/src/__tests__/KeybindingHint.test.tsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react'
```

Comment:
(not blocking) You can move these inside `packages/react/src/KeyboardHint/__tests__`, that's the preferred location